### PR TITLE
Knex merge

### PR DESCRIPTION
**Pull request by abrden**

### CODE REVIEW

File: src/handlers/db.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+const logger = require('../utils/logger');
+const models = require('../models/index');
+const db = require('../database/');
+
+const tables = {
+  users: models.users,
+  admins: models.admins,
+};
+
+const findAllEntries = (tableName) => {
```
Comment (manuporto) [inline src/handlers/db.js]: Use `function` instead or arrow functions

File: src/handlers/response.js [hunk 2]
```diff
@@ -0,0 +1,189 @@
+const constants = require('./../routes/constants.json');
+const logger = require('../utils/logger');
+const promisify = require('promisify-node');
+const amanda = require('amanda');
+
+const jsonSchemaValidator = amanda('json');
+
+const internalServerError = (reason, response) => {
```
Comment (manuporto) [inline src/handlers/response.js]: Use `function` instead of arrow functions

File: src/handlers/response.js [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+  logger.info('Successful user creation');
+  response.status(201).json(user);
+};
+
+const successfulUserUpdate = (user, response) => {
+  logger.info('Successful user update');
+  response.status(200).json(user);
+};
+
+const successfulUserDeletion = (response) => {
```
Comment (manuporto) [inline src/handlers/response.js]: No need to use parenthesis when using a one argument arrow function

File: src/handlers/response.js (hunk 2 above)
Comment (abrden) [inline src/handlers/response.js]: no viejo. es6, esta todo pasado a arrow functions

File: src/handlers/response.js (hunk 3 above)
Comment (abrden) [inline src/handlers/response.js]: true dat

File: src/handlers/db.js (hunk 1 above)
Comment (abrden) [inline src/handlers/db.js]: how about no (?

File: src/handlers/response.js (hunk 3 above)
Comment (abrden) [inline src/handlers/response.js]: pd el linter te obliga a ponerle parentesis

**Pull request merged**